package/MDAnalysis/topology/TOPParser.py

```diff
@@ -563,9 +586,16 @@ def parsesection_mapper(self, numlines, mapper):
             A list of all entries in a given parm7 section
         """
         section = []
-        y = next(self.topfile).strip("%FORMAT(")
-        y.strip(")")
-        x = FORTRANReader(y)
+
+        def get_fmt(file):
```

Comment:
Add a comment to state what this does.

I _think_ it will keep reading until it hits a %FORMAT line with the idea of skipping %COMMENT lines.

However, if this is the case, then why not make it more explicit because then we could just fail if we find neither %COMMENT nor %FORMAT. I think these should be the only to valid options in the format, so then being explicit is safer and clearer than being implicit.

Reply:
Done.

Reply:
It's still not immediately clear to the reader as to what this does. Can the comment be a straight up one line docstring on "what does get_fmt aim to do"?

Reply:
Done.